### DISCUSSION TITLE
Who is bob?

### DISCUSSION THREAD

**Discussion by toolness** [mysticsymbolic/mysticsymbolic.github.io#181] [General]
This "bob" person seems to be a recognizable character, from the Twitter threads I've seen:

https://github.com/mysticsymbolic/mysticsymbolic.github.io/blob/main/assets/symbols/bob.svg

Is there a story behind this character @ninapaley?

**Comment by ninapaley** [comment]
Bob is the icon of the Church of the Subgenius: https://en.wikipedia.org/wiki/Church_of_the_SubGenius

https://www.google.com/search?q=bob+subgenius&sxsrf=ALeKk03yf2oq1quCFAuFLwJ2z7t-RG9hfQ:1624195281144&source=lnms&tbm=isch&sa=X&ved=2ahUKEwjP15z2pqbxAhVNvJ4KHZ7OAe0Q_AUoAXoECAEQAw&biw=1625&bih=862

Since I had a few other religious symbols, like a cross and star of david, I thought JR "Bob" Dobbs sould be in there too.

**Reply by toolness** [reply]
Heh heh, that's cool!  Thanks for the explanation.
